Cargo.toml

```diff
@@ -31,7 +31,7 @@ exclude = ["examples/basic", "examples/test_framework"]
 
 [workspace.package]
 # don't forget to update `workspace.dependencies` below
-version = "0.11.3"
+version = "0.11.4"
```

Comment:
Please don't bump the version here. You can request a new release in the comments, but the decision when to releases is up to the maintainers. Also, we doing the release should be done in a separate PR anyway because it requires additional steps (e.g. changelog creation).